server/game/cards/02_SHD/units/SurvivorsGauntlet.ts

```diff
@@ -0,0 +1,45 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { RelativePlayer, TargetMode, WildcardCardType, WildcardRelativePlayer, WildcardZoneName } from '../../../core/Constants';
+
+export default class SurvivorsGauntlet extends NonLeaderUnitCard {
+    protected override getImplementationId() {
+        return {
+            id: '1503633301',
+            internalName: 'survivors-gauntlet',
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addTriggeredAbility({
+            title: 'Attach an upgrade on a unit to another eligible unit controlled by the same player',
+            when: {
+                onCardPlayed: (event, context) => event.card === context.source,
+                onAttackDeclared: (event, context) => event.attack.attacker === context.source,
+            },
+            targetResolvers: {
+                chooseUpgrade: {
+                    mode: TargetMode.Single,
```

Comment:
This shouldn't be needed, I believe it's the default (same for the other resolver)